docs/CHANGELOG.rst

```diff
@@ -15,11 +15,13 @@ Added
 * Add Resolwe API to test url configuration
 * Add TransactionBioProcessTestCase to support writing tests for
   processes which use resdk
+* Add ``genialis/chipseq`` docker image
```

Comment:
We already use [genialis/iclip](https://hub.docker.com/r/genialis/iclip/). I followed the same naming schema. @tjanez ?